crates/cargo-test-support/src/registry.rs

```diff
@@ -335,7 +336,7 @@ impl Package {
 
     /// Adds an entry in the `[features]` section.
     pub fn feature(&mut self, name: &str, deps: &[&str]) -> &mut Package {
-        let deps = deps.iter().map(|s| s.to_string()).collect();
+        let deps = deps.iter().map(|s| (*s).to_string()).collect();
```

Comment:
I think these `(*s)` changes hinder readability for a micro-optimization that I don't think matters, especially in test code.